app/views/user/_actions.haml

```diff
@@ -36,6 +36,10 @@
             %a.dropdown-item{ href: '#', data: { target: '#modal-ban', toggle: :modal } }
               %i.fa.fa-ban
               = t 'views.actions.ban'
+        - if current_user.mod? && user != current_user
+          %a.dropdown-item{ href: mod_inbox_index_path(user: user.screen_name) }
+            %i.fa.fa-inbox
+            View inbox
```

Comment:
You can add a locale for `views.actions.` in `en.yml` here, I'll migrate it over in a PR when I refactor all user profile translations

Reply:
e024d8b